code/modules/organs/subtypes/machine.dm

```diff
@@ -62,26 +62,101 @@
 	desc = "A small, powerful cell for use in fully prosthetic bodies."
 	icon = 'icons/obj/power.dmi'
 	icon_state = "scell"
-	organ_tag = "cell"
+	organ_tag = BP_CELL
 	parent_organ = BP_CHEST
-	vital = 1
-	var/emp_counter = 0
+	vital = TRUE
```

Comment:
It being both vital and replaceable is kind of weird.

Reply:
It makes sense, though. Can't do anything w/o a cell.

Reply:
I don't think it _really_ matches with what the `var/vital` is meant to be used for. I would prefer if the IPCs had an *unpowered* state that was different from `CONSCIOUS`, `UNCONSCIOUS` and `DEAD` -- But that would be a nightmare to do. So nevermind.